NeuroPilot/ScoutPatches.cs

```diff
@@ -0,0 +1,118 @@
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using HarmonyLib;
+using NeuroPilot.Actions;
+using NeuroSdk.Actions;
+using UnityEngine;
+
+namespace NeuroPilot
+{
+
+    [HarmonyPatch]
+    public class ScoutPatches
+    {
+        public static HashSet<SurveyorProbe> surveyorProbes = new();
+        public static Color surveyorProbeColor = Color.white;
+        public static int surveyorProbeIntensity = 1;
+        public static ProbeLauncher probeLauncher = null;   // Store the launcher to be able to take Snapshots
+
+        [HarmonyPostfix, HarmonyPatch(typeof(ProbeLauncher), nameof(ProbeLauncher.LaunchProbe))]
+        public static void ProbeLauncher_LaunchProbe_Postfix(ProbeLauncher __instance)
+        {
+            if (NeuroPilot.instance.ModHelper.Config.GetSettingsValue<bool>("Scout Launcher (Neuro)"))
```

Comment:
This is more personal preference but these config setting retrievals make the lines very long and create a lot of potential for typos. They could be moved behind some static property getters to be more succinct and avoid duplicate use of bare strings.